packages/tools/src/comparison/compare-runs.ts

```diff
@@ -0,0 +1,257 @@
+  }
+  assert.sameMembers(inputFilesLists[0], inputFilesLists[1])
+
+  // Compare the model-specific input files.
+  for (const fileName of inputFilesLists[0]) {
+    // same file names in both lists
+    if ([RUNNER_INPUT_FILENAME, MODEL_INPUT_FILENAME].includes(fileName)) {
+      // Skip these, as we compare the JSON elsewhere.
+      continue
+    }
+    logger.info('Comparing input files named %s', fileName)
+    const filePaths = inputDirs.map(dir => path.join(dir, fileName))
+    const fileContents = await Promise.all(
+      filePaths.map(async filePath =>
+        fs.promises.readFile(filePath, { encoding: 'utf8' })
+      )
+    )
+    if (isCovidSimInputFile(fileName)) {
+      const parameterObjects = fileContents.map(parse)
+      assert.deepEqual(
+        parameterObjects[0],
+        parameterObjects[1],
+        'CovidSim parameter objects differ between runs'
+      )
+    } else {
+      assert.equal(
+        fileContents[0],
+        fileContents[1],
+        `Input files named ${fileName} differ between runs`
+      )
+    }
+  }
+}
+
+async function getModelOutput(tmpDir: string, i: number) {
+  const unzippedOutput = getUnzippedOutputsPath(tmpDir, i)
+  const modelOutputPath = path.join(unzippedOutput, MODEL_OUTPUT_FILENAME)
+  logger.info('Reading model output %d from %s', i, modelOutputPath)
+  enforceOutputSchema(modelOutputPath)
+  const modelOutputData = await fs.promises.readFile(modelOutputPath, 'utf8')
+  return JSON.parse(modelOutputData) as output.ModelOutput
+}
+
+/** Unzips an archive from `zippedPath`, writing its contents to `unzippedPath`. */
+async function unzip(zippedPath: string, unzippedPath: string) {
+  const readStream = fs.createReadStream(zippedPath)
+  const unzipStream = unzipper.Extract({
+    path: unzippedPath,
+    forceStream: true,
+  })
+  await new Promise((resolve, reject) => {
+    readStream.on('error', reject)
+    unzipStream.on('error', reject)
+    unzipStream.on('close', resolve)
+    readStream.pipe(unzipStream)
+  })
+}
+
+function compareModelOutputs(modelOutputs: output.ModelOutput[]) {
+  logger.info('Comparing run outputs')
+  assert.deepEqual(
+    modelOutputs[0].metadata,
+    modelOutputs[1].metadata,
+    'Model output metadata do not match'
+  )
+  assert.equal(
+    modelOutputs[0].time.t0,
+    modelOutputs[1].time.t0,
+    'Starting timestamps do not match'
+  )
+  assert.sameOrderedMembers(
+    modelOutputs[0].time.extent,
+    modelOutputs[1].time.extent,
+    'Simulation extents do not match'
+  )
+  assert.sameOrderedMembers(
+    modelOutputs[0].time.timestamps,
+    modelOutputs[1].time.timestamps,
+    'Simulation timestamps do not match'
+  )
+
+  const sum = (xs: number[]) => xs.reduce((a, b) => a + b, 0)
+  const last = (xs: number[]) => xs[xs.length - 1]
+  const totalDeaths = modelOutputs.map(o => sum(o.aggregate.metrics.incDeath))
+
+  const CUMULATIVE_DEATHS_DIFFERENCE_THRESHOLD = 1000
+  const CUMULATIVE_CASES_DIFFERENCE_THRESHOLD = 1000
```

Comment:
This seems low based on the wild swings I've seen, but I really don't know. I guess let's try this and find out.

Reply:
This was an entirely arbitrary choice (and 100 was too low for the mismatched runs we were comparing). Happy to change.